docs/book/driver-deployment/upgrade.md

```diff
@@ -21,12 +83,77 @@ kubectl delete -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-
 
 Wait for vSphere CSI Controller Pod and vSphere CSI Nodes Pods to be completely deleted.
 
-Install `v2.0.1` driver
+### Install `v2.3.0` driver
 
-```bash
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/release-2.0/manifests/v2.0.1/vsphere-67u3/rbac/vsphere-csi-controller-rbac.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/release-2.0/manifests/v2.0.1/vsphere-67u3/deploy/vsphere-csi-controller-deployment.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/release-2.0/manifests/v2.0.1/vsphere-67u3/deploy/vsphere-csi-node-ds.yaml
-```
+Refer to the [release notes](../releases/v2.3.0.md) of `v2.3.0` for detail.
+
+#### Pre-requiresites for `v2.3.0`
+
+`v2.3.0` release of the driver requires DNS forwarding configuration in CoreDNS ConfigMap to help resolve vSAN file share hostname.
+
+Modify the CoreDNS ConfigMap and add the conditional forwarder configuration:
+
+   ```bash
+   kubectl -n kube-system edit configmap coredns
+   ```
+
+Output:
+
+   ```bash
+   .:53 {
+    errors
+    health {
+       lameduck 5s
+    }
+    ready
+    kubernetes cluster.local in-addr.arpa ip6.arpa {
+       pods insecure
+       fallthrough in-addr.arpa ip6.arpa
+       ttl 30
+    }
+    prometheus :9153
+    forward . /etc/resolv.conf {
+       max_concurrent 1000
+    }
+    cache 30
+    loop
+    reload
+    loadbalance
+   }
+   vsanfs-sh.prv:53 {
```

Comment:
We have to add this to installation docs as well right?
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/a9250f6540080be9868488200b95447d983cea3a/docs/book/driver-deployment/installation.md#install

Reply:
we are adding this at https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html